packages/template-cli/src/__tests__/domain/ssr.bootstrap-config.json

```diff
@@ -4,7 +4,7 @@
   "platform": "aks",
   "deployment": "azdevops",
   "advanced_config": true,
-  "create_config": true,
+  "create_config": false,
```

Comment:
Should this be false?

Reply:
it's just a mock input  for a test. it tests both - with and without config enabled